File: TOMIMSV4/Entity/Mantenimientos/Propietario/Reglas/clsBePropietario_reglas_det.vb
Public Class clsBePropietario_reglas_det
    Implements ICloneable
    Implements IDisposable

    Public Property IdReglaPropietarioDet() As Integer = 0
    Public Property IdReglaPropietarioEnc() As Integer = 0
    Public Property IdDestinatarioPropietario() As Integer = 0
    Public Property User_agr() As String = ""
    Public Property Fec_agr() As Date = Date.Now
    Public Property User_mod() As String = ""
    Public Property Fec_mod() As Date = Date.Now
    Public Property Activo() As Boolean = False

    Sub New()
    End Sub
    Sub New(ByRef IdReglaPropietarioDet As Integer, ByVal IdReglaPropietarioEnc As Integer, ByVal IdDestinatarioPropietario As Integer, ByVal user_agr As String, ByVal fec_agr As Date, ByVal user_mod As String, ByVal fec_mod As Date, ByVal activo As Boolean)
        Me.IdReglaPropietarioDet = IdReglaPropietarioDet
        Me.IdReglaPropietarioEnc = IdReglaPropietarioEnc
        Me.IdDestinatarioPropietario = IdDestinatarioPropietario
        Me.User_agr = User_agr
        Me.Fec_agr = Fec_agr
        Me.User_mod = User_mod
        Me.Fec_mod = Fec_mod
        Me.Activo = Activo
    End Sub
    Public Function Clone() As Object Implements System.ICloneable.Clone
        Return MyBase.MemberwiseClone()
    End Function
End Class

File: WMSWebAPI/Entity/Propietario/clsBePropietario_reglas_det.cs
using System.ComponentModel;
using System.ComponentModel.DataAnnotations.Schema;

namespace WMSWebAPI.Entity.Propietario
{
    public class clsBePropietario_reglas_det : ICloneable
    {
        [Column("IdReglaPropietarioDet")]
        [DisplayName("IdReglaPropietarioDet")]
        public int IdReglaPropietarioDet { get; set; } = 0;

        [Column("IdReglaPropietarioEnc")]
        [DisplayName("IdReglaPropietarioEnc")]
        public int IdReglaPropietarioEnc { get; set; } = 0;

        [Column("IdDestinatarioPropietario")]
        [DisplayName("IdDestinatarioPropietario")]
        public int IdDestinatarioPropietario { get; set; } = 0;

        [Column("user_agr")]
        [DisplayName("user_agr")]
        public string User_agr { get; set; } = "";

        [Column("fec_agr")]
        [DisplayName("fec_agr")]
        public DateTime Fec_agr { get; set; } = DateTime.Now;

        [Column("user_mod")]
        [DisplayName("user_mod")]
        public string User_mod { get; set; } = "";

        [Column("fec_mod")]
        [DisplayName("fec_mod")]
        public DateTime Fec_mod { get; set; } = DateTime.Now;

        [Column("activo")]
        [DisplayName("activo")]
        public bool Activo { get; set; } = false;

        public clsBePropietario_reglas_det() { }

        public object Clone()
        {
            return MemberwiseClone();
        }
    }
}